src/core/renderController.ts

```diff
@@ -18,10 +18,10 @@ module Plottable {
    * ```
    */
   export module RenderController {
-    var _componentsNeedingRender = new Utils.Set<Component>();
-    var _componentsNeedingComputeLayout = new Utils.Set<Component>();
-    var _animationRequested = false;
-    var _isCurrentlyFlushing = false;
+    var componentsNeedingRender = new Utils.Set<Component>();
+    var componentsNeedingComputeLayout = new Utils.Set<Component>();
+    var animationRequested = false;
+    var isCurrentlyFlushing = false;
```

Comment:
They are not exposed to the user.  There's no way for them to access these vars.